src/workflows-app/RunDetails.js

```diff
@@ -151,6 +157,33 @@ export const BaseRunDetails = (
     [signal, workspaceId]
   );
 
+  const inProgressElement = (workflow) => {
```

Comment:
Nit: for convention, suggest naming functions that return React elements `render...`.

```suggestion
  const renderInProgressElement = (workflow) => {
```